lib/memo_wise.rb

```diff
@@ -187,12 +187,17 @@ def klass.extended(base)
           # hash key is just the method name.
           klass.module_eval <<-END_OF_METHOD, __FILE__, __LINE__ + 1
             def #{method_name}
-              _memo_wise_output = @_memo_wise[#{index}]
-              if _memo_wise_output || @_memo_wise_sentinels[#{index}]
-                _memo_wise_output
+              if @_memo_wise_sentinels[#{index}]
```

Comment:
Since this change will impact performance slightly, would you mind updating [the `README.md`'s benchmarks](https://github.com/panorama-ed/memo_wise#benchmarks) to use the numbers from [our CI run on your branch](https://github.com/panorama-ed/memo_wise/runs/4215740857?check_suite_focus=true)? For convenience, here they are:

```markdown
|Method arguments|`Dry::Core` (0.7.1)|`Memery` (1.4.0)|
|--|--|--|
|`()` (none)|1.42x|17.84x|
|`(a)`|2.48x|11.48x|
|`(a, b)`|0.46x|2.05x|
|`(a:)`|2.18x|20.50x|
|`(a:, b:)`|0.48x|4.22x|
|`(a, b:)`|0.48x|4.08x|
|`(a, *args)`|0.90x|1.97x|
|`(a:, **kwargs)`|0.80x|3.02x|
|`(a, *args, b:, **kwargs)`|0.61x|1.54x|
```

```markdown
|Method arguments|`DDMemoize` (1.0.0)|`Memoist` (0.16.2)|`Memoized` (1.0.2)|`Memoizer` (1.0.3)|
|--|--|--|--|--|
|`()` (none)|33.90x|3.44x|1.56x|4.03x|
|`(a)`|24.56x|17.02x|12.94x|14.91x|
|`(a, b)`|3.14x|2.35x|1.84x|2.03x|
|`(a:)`|34.42x|28.14x|23.83x|25.26x|
|`(a:, b:)`|5.13x|4.28x|3.77x|3.97x|
|`(a, b:)`|4.83x|4.08x|3.50x|3.66x|
|`(a, *args)`|3.03x|2.25x|1.95x|1.95x|
|`(a:, **kwargs)`|2.90x|2.44x|2.14x|2.24x|
|`(a, *args, b:, **kwargs)`|2.05x|1.77x|1.65x|1.64x|
```

Reply:
Updated README with these numbers.